src/domWrapper.ts

```diff
@@ -137,4 +140,25 @@ export class DOMWrapper<ElementType extends Element>
 
     return new DOMWrapper(parentElement).trigger('change')
   }
+
+  async trigger(
+    eventString: DomEventName,
```

Comment:
While reviewing I realized that with TS 4.1+, we could even type and have autocompletion for things like `trigger('click.prevent')` with template literal types. But let's keep this for a followup PR 😅

Reply:
agreed, I'll create a PR for this once, this is merged.